vendor/github.com/open-policy-agent/cert-controller/pkg/rotator/rotator.go

```diff
@@ -163,6 +163,11 @@ type CertRotator struct {
 	caNotInjected          chan struct{}
 }
 
+// NeedLeaderElection is false as this runnable can run in standby mode without leader lock
+func (cr *CertRotator) NeedLeaderElection() bool {
+	return false
+}
+
```

Comment:
This seems a bit weird - what happens if multiple instances try to create conflicting certs for the same webhook at the same time? I wonder if this will _normally_ work but not _always_. Do you know?

Otherwise, an alternative option here might be to leave the runnables as requiring leader election, but then provide a function for the followers to determine when it's safe to start. E.g. maybe modify `ensureCertsMounted()` so that it's a standalone function that you can call (or just factor it out so as to preserve the existing interface), and it simply blocks until the certs exist. wdyt?

Either way, we should add this to the main cert-controller as well. Can you please file an issue to do so? Thanks!